packages/create-atomic/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@coveo/create-atomic",
+  "description": "Coveo Atomic Generator",
+  "version": "1.18.0",
```

Comment:
I think we should not publish this package yet, we should add https://docs.npmjs.com/cli/v7/configuring-npm/package-json#private for now.